pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools>=45", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "cell_feature_data"
+version = "1.0.0"
+description = "A Python package for creating a dataset for the Cell Feature Explorer."
+keywords = []
+readme = "README.md"
+requires-python = ">=3.8"
+classifiers = [
+"Development Status :: 4 - Beta",
+"Natural Language :: English",
+"Programming Language :: Python :: 3.7",
+"Programming Language :: Python :: 3.8",
+"Programming Language :: Python :: 3.9",
+"Programming Language :: Python :: 3.10",
+"Programming Language :: Python :: 3.11",
```

Comment:
I suggest adding 3.12 to this list and removing 3.7. (line 11 already says it requires Python >= 3.8)   Even Python 3.8 is nearing its end-of-life: https://devguide.python.org/versions/